commands/inventory/add_agent_mongodb_exporter.go

```diff
@@ -101,4 +103,5 @@ func init() {
 	AddAgentMongodbExporterC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddAgentMongodbExporter.SkipConnectionCheck)
 	AddAgentMongodbExporterC.Flag("tls", "Use TLS to connect to the database").BoolVar(&AddAgentMongodbExporter.TLS)
 	AddAgentMongodbExporterC.Flag("tls-skip-verify", "Skip TLS certificates validation").BoolVar(&AddAgentMongodbExporter.TLSSkipVerify)
+	AddAgentMongodbExporterC.Flag("use_vmagent", "When false, pull model is used. By default, its true").Default("true").BoolVar(&AddAgentMongodbExporter.UseVMAgent)
```

Comment:
The second problem that I see here, it's related to true by default https://github.com/alecthomas/kingpin/issues/243.
There is nothing in help how to disable it.

Reply:
old code